### ISSUE TITLE
misleading information about domestic & international calling plan

### ISSUE THREAD

**Issue by fs79** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#652]
Hi,
would you describe better that "domestic & international" plan means 1.200 minutes domestic & 600 minutes international are included.
The sentence "600 minute for Domestic and International Calling Plans" leads to 600 minutes for both, domestic & international, which is not true.

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 02354090-59c2-297d-2415-70a2cbdd15c4
* Version Independent ID: c6dfc9e6-e2f8-2f3c-6172-e15f6311d0aa
* Content: [Availability in Germany](https://docs.microsoft.com/en-us/SkypeForBusiness/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-germany#calling-plans)
* Content Source: [Skype/SfbOnline/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-germany.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbOnline/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-germany.md)
* Service: **skype-for-business-online**
* GitHub Login: @tonysmit
* Microsoft Alias: **tonysmit**

**Comment by AndreaBarr** [comment]
@fs79 Thank you for this feedback

We appricate you pointing this out. I am sending this over to the Teams/Skype for Business team to get addressed properly. 

I was able to get this into the backlog for the Teams/Skype for Business team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
